docs/source/index.rst

```diff
@@ -48,7 +48,7 @@ can make ER research and development easier for ourselves and others.
 Quickstart
 ----------
 
-pseudopeople requires a version of `Python <https://www.python.org/>`_ between 3.7 and 3.10 (inclusive) to be installed.
+pseudopeople requires a version of `Python <https://www.python.org/>`_ between 3.7 and 3.11 (inclusive) to be installed.
```

Comment:
Looks like 3.7->3.8 got missed here. Since you're in the area, we should fix it now.